resources/js/features/games/components/GameAchievementSetsContainer/GameAchievementSet/GameAchievementSetProgress/MasteredProgressIndicator/MasteredProgressIndicator.tsx

```diff
@@ -194,6 +234,48 @@ const FloatableContent: FC<FloatableContentProps> = ({ achievements }) => {
         </div>
       ) : null}
 
+      {completionDate ? (
+        <>
+          <BaseSeparator className="my-2" />
+
+          <div className="flex flex-col gap-0.5">
+            <div className="flex justify-between text-2xs">
+              <p>{isMastered ? t('Mastered on') : t('Completed on')}</p>
+              <p className="font-medium">{formatDate(completionDate, 'll')}</p>
+            </div>
+
+            {timeTaken ? (
+              <div className="flex justify-between text-2xs">
+                <p>{isMastered ? t('Time to master') : t('Time to complete')}</p>
+                <p className="font-medium">{formatDuration(timeTaken)}</p>
```

Comment:
Is there some point where we shouldn't report seconds any more?

<img width="213" height="236" alt="Image" src="https://github.com/user-attachments/assets/af6e519a-88fc-4b24-90e2-7df56d0b4355" />